qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -238,7 +238,26 @@ def selection(self, html=False):
         return self._widget.selectedText()
 
     def follow_selected(self, *, tab=False):
-        log.stub()
+        def _follow_selected_cb(js_elem):
+            print(js_elem)
+            if js_elem is not None:
+                elem = webengineelem.WebEngineElement(js_elem, tab=self._tab)
+                if tab:
+                    click_type = usertypes.ClickTarget.tab
+                else:
+                    click_type = usertypes.ClickTarget.normal
+
+                # Check to see if we have a real link, to follow before clicking
+                href_tags = ['a', 'area', 'link']
```

Comment:
When issuing clicks, it can get a little messy if we blindly click the selected element. I want to check to see if the element is a link before clicking it, is there a better way to do this? I think the cleanest way to do this is via a new method in webelem.py (click_if_link maybe?) Do you think that's a good idea? If so I'll go ahead and do that!